client/src/app/budgets/budget/budget.component.html

```diff
@@ -1,4 +1,5 @@
 <div class="card text-center" style="margin-bottom:20px;">
+    <p>{{budgetTypeName}}</p>
```

Comment:
This change breaks the layout (see the white row in the screens provided by you)